okd/src/microshift-okd-multi-build.Containerfile

```diff
@@ -0,0 +1,56 @@
+FROM quay.io/centos-bootc/centos-bootc:stream9 as builder
+
+	
+ARG OKD_REPO=quay.io/okd/scos-release
+ARG OKD_VERSION_TAG=4.17.0-0.okd-scos-2024-08-21-100712
+ARG REPO_DIR=/src/_output/rpmbuild/RPMS/
+ENV USER=microshift
+ENV HOME=/microshift
+ENV GOPATH=/microshift
+ENV GOMODCACHE=/microshift/.cache
+
+# Adding non-root user for building microshift
+run useradd -m -s /bin/bash -p "$(openssl passwd -1 redhat)" microshift
+run echo -e 'microshift\tALL=(ALL)\tNOPASSWD: ALL' | sudo tee /etc/sudoers.d/microshift
+RUN mkdir /microshift
+copy . /src 
+RUN chown -R 1000:1000 /microshift /src
```

Comment:
Why are we using hardcoded uid and gid? Can we use `microshift:microshift` instead?